code/modules/organs/limb_objects.dm

```diff
@@ -1,8 +1,31 @@
+#define LIMB_REATTACH_TIME 15 MINUTES
 
-obj/item/limb
+/obj/item/limb
 	icon = 'icons/mob/human_races/r_human.dmi'
-
-obj/item/limb/New(loc, mob/living/carbon/human/H)
+	var/amputation_time
+	var/mob/living/carbon/human/owner
```

Comment:
Can this be carbon? Monkeys and other things have limbs.

Reply:
As of now only humans seems to have limbs, monkeys don't.

Reply:
I see. It's something that we'll want in the future, though, most likely.

Reply:
when they have limbs this can be changed to carbon.